android/src/main/java/com/bitmovin/player/reactnative/RNPlayerView.kt

```diff
@@ -98,6 +98,8 @@ class RNPlayerView(val context: ReactApplicationContext) :
     View.OnLayoutChangeListener,
     RNPictureInPictureDelegate {
 
+    data class PictureInPictureConfig(val isEnabled: Boolean)
```

Comment:
This should maybe be moved to the RNPictureInPictureHandler.kt file.

Reply:
fixed in 90b13a1